test/helpers/gas-usage.ts

```diff
@@ -65,21 +77,33 @@ const MAX_GAS_PER_GROUP: any = {
   [GasGroup.EXECUTE_OPERATOR_FEE]: 52000,
   [GasGroup.REDUCE_OPERATOR_FEE]: 51900,
 
-  [GasGroup.REGISTER_VALIDATOR_EXISTING_CLUSTER]: 202000,
+  [GasGroup.REGISTER_VALIDATOR_EXISTING_CLUSTER]: 201200,
```

Comment:
@andrew-blox Refactoring the code gets savings on the gas costs, around 1500 gas per GasGroup, but as this version uses `ssv-keys` to generate the shares, all values are increased.